webui/react/src/shared/utils/service.ts

```diff
@@ -13,7 +13,7 @@ export const isAuthFailure = (e: any, supportExternalAuth = false): boolean => {
   const status = getResponseStatus(e) ?? 0;
   const authFailureStatuses = [ 401 ];
   if (supportExternalAuth) authFailureStatuses.push(500);
-  return authFailureStatuses.includes(status);
+  return authFailureStatuses.includes(status) || (status >= 400 && status < 500);
```

Comment:
could we just have `(supportExternalAuth && status === 500) || (status >= 400 && status < 500)`?